src/main/resources/crafter/studio/studio-config.yaml

```diff
@@ -889,6 +889,20 @@ studio.marketplace.plugin.mapping:
   - sources: sources
 # The patter to use for loading plugins installed from the marketplace
 studio.marketplace.plugin.basePath: '/config/studio/plugins/js/${id}/${type}/${name}'
+# Mapping used to wire plugin files into the site configuration
+studio.marketplace.plugin.wire.mapping:
+  sidebar:
+    module: studio
+    path: ui.xml
+    template: crafter/studio/marketplace/wiring/ui.xslt
```

Comment:
What happens when we have more widgets wired within:
* ui.xml but not sidebar (like the upcoming context nav, or megamenu)
* global configuration of the whole system (new ui.xml file in global repo)